lib/caps/ADL/iHD

```diff
@@ -91,7 +91,7 @@ caps = dict(
     # colorspace conversion
     csc         = dict(
       ifmts = ["NV12", "YV12", "I420", "P010", "YUY2", "UYVY", "Y210", "AYUV", "Y410", "BGRA"],
-      ofmts = ["NV12", "YV12", "I420", "P010", "YUY2",         "Y210", "AYUV", "Y410", "BGRA"],
+      ofmts = ["NV12", "YV12", "I420", "P010", "YUY2",         "Y210", "AYUV", "Y410", "BGRA", "BGRX"],
```

Comment:
Is BGRX supported for input, too (i.e. ifmts)?

Reply:
> Is BGRX supported for input, too (i.e. ifmts)?

Based on gst-inspect for gst-msdk, BGRX is supported, and quick run 1-2 cmdline, it worked, but still not detail manual testing yet.  On-going to enable more VPP CSC feature into vaapi-fits after manual testing